extensions/amp-story/1.0/amp-story.js

```diff
@@ -775,6 +776,11 @@ export class AmpStory extends AMP.BaseElement {
 
     this.getViewport().onResize(debounce(this.win, () => this.onResize(), 300));
     this.installGestureRecognizers_();
+
+    // Install parallax handlers if opt-in is detected on a mobile environment
+    if (!this.isDesktop_() && this.element.hasAttribute('parallax-fx')) {
```

Comment:
The desktop UI triggers based on the screen size, but some tablets could still trigger it (like iPads). I think we could change this optin to something like `(isAndroid || isIOs) && hasParallaxAttribute` (see [platform-impl](https://github.com/ampproject/amphtml/blob/master/src/service/platform-impl.js#L35-L49))

Wdyt?